src/frontend/components/Button/ButtonLayout/index.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useRef, useState, useLayoutEffect, useContext } from 'react';
+import { Box, Text, ResponsiveContext } from 'grommet';
+import ResizeObserver from 'resize-observer-polyfill';
+import styled from 'styled-components';
+
+import { SvgProps } from 'components/SVGIcons';
+import { Nullable } from 'utils/types';
+
+interface ResponsiveProps {
+  screenSize: string;
+}
+
+const IconBox = styled(Box)`
+  display: flex;
+  height: ${({ screenSize }: ResponsiveProps) =>
+    screenSize === 'small' ? '60%' : '80%'};
+  margin: ${({ screenSize }: ResponsiveProps) =>
+    screenSize === 'small' ? '0 6px' : '0 8px'};
+  margin-bottom: 6px;
+  min-height: 0;
+  position: relative;
+  width: 100%;
```

Comment:
I think this property can be passed as a prop of the IconBox component

Reply:
c72b659